README.md

```diff
@@ -15,6 +15,7 @@ The following devices have been tested with EdgeX:
 
 ## How does the device service work?
 - The device service ONLY works on Linux system.
+- The Linux kernel version is restricted to v5.10.x.
```

Comment:
That is the minimum, not the only minor version.
```suggestion
- The device service ONLY works on Linux with kernel v5.10 or higher.
```

Reply:
Maybe their libc-dev wasn't built against 5.11?

Reply:
The APIs suppose to be backward compatible. Both Alpine 3.15 (used as the docker image base) and Ubuntu 22.04 (use as snap base) use kernel 5.15 right now, on which we can build and run the application without issues.